src/NexusMods.App.UI/Controls/Buttons/StandardButtonTheme.axaml

```diff
@@ -0,0 +1,370 @@
+                    <buttons:StandardButton Theme="{DynamicResource Tertiary}" />
+                </StackPanel>
+
+                <StackPanel Orientation="Vertical">
+                    <TextBlock>Tertiary</TextBlock>
+                    <buttons:StandardButton Theme="{DynamicResource Tertiary}" />
+                    <buttons:StandardButton Theme="{DynamicResource Tertiary}" />
+                </StackPanel>
+            </StackPanel>
+
+            <StackPanel Orientation="Horizontal">
+                <StackPanel Classes="WithIcon" Orientation="Vertical">
+                    <TextBlock>Success + Icon</TextBlock>
+                    <buttons:StandardButton Theme="{DynamicResource Success}" />
+                    <buttons:StandardButton Theme="{DynamicResource Success}" />
+                </StackPanel>
+
+                <StackPanel Classes="WithImage" Orientation="Vertical">
+                    <TextBlock>Success + Image</TextBlock>
+                    <buttons:StandardButton Theme="{DynamicResource Success}" />
+                    <buttons:StandardButton Theme="{DynamicResource Success}" />
+                </StackPanel>
+
+                <StackPanel Orientation="Vertical">
+                    <TextBlock>Success</TextBlock>
+                    <buttons:StandardButton Theme="{DynamicResource Success}" />
+                    <buttons:StandardButton Theme="{DynamicResource Success}" />
+                </StackPanel>
+            </StackPanel>
+        </StackPanel>
+    </Design.PreviewWith>
+
+    <ControlTheme x:Key="Standard" TargetType="buttons:StandardButton">
+        <ControlTheme.Resources>
+            <icons1:IconTypeConverter x:Key="IconTypeConverter" />
+            <converters:TextCaseConverter x:Key="TextCaseConverter" />
+        </ControlTheme.Resources>
+
+        <Setter Property="Height" Value="34" />
+        <Setter Property="CornerRadius" Value="4" />
+
+        <Setter Property="Template">
+            <ControlTemplate>
+                <Border x:Name="PART_Border"
+                        Height="{TemplateBinding Height}"
+                        Background="{TemplateBinding Background}"
+                        CornerRadius="{TemplateBinding CornerRadius}">
+                    <StackPanel x:Name="PART_Panel">
+                        <icons:Icon x:Name="PART_Icon"
+                                    Foreground="{TemplateBinding Foreground}"
+                                    Value="{TemplateBinding IconType, Converter={StaticResource IconTypeConverter}}" />
+                        <Image x:Name="PART_Image"
+                               Source="{TemplateBinding Property=Image}" />
+                        <TextBlock x:Name="PART_TextBlock"
+                                   Foreground="{TemplateBinding Foreground}"
+                                   Text="{TemplateBinding Text, Converter={StaticResource TextCaseConverter}, ConverterParameter=upper}" />
```

Comment:
In the interest of efficiency, it might be a good idea to reference the converters via `x:Static` instances, given that we might reuse them elsewhere.

It should also speed up instantiation of the control, given we avoid a dictionary lookup (per control), and reduce memory use.

-----

Might also be handy to make a converter for each case (makes usage easier from xaml as use doesn't need to know the 'magic' parameters). Should be doable fairly easy with abstract class; and abstract method (which parent implements) to do the conversion. Then make that sealed and you got both convenience and efficiency at around same LoC.